### PR TITLE
[SEMI-MODULAR] Fixes unique AI, Adds Crewsimov, Fixes default config file

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

So basically an AI lawset PR (two of them) were merged blindly, and merge conflict markers were left in them. That is fixed now, and I've reset the fallback lawset to safeguard. Additionally, I've overrode Asimov with crewsimov since the whole reason TG runs Asimov as because humans are command. We are not TG, and this allows us to be a bit more sensible. 

Also, this modifies the "CC DOESNT WANT US TO CHANGE IT" nonsense. We're not TGstation, and I am sure they wouldn't care about the AI's laws being changed either. We shouldn't be importing TG's memes, cause the players here treat this seriously and will reeee if you so much as think about changing the AI's laws. 

## How This Contributes To The Skyrat Roleplay Experience

Let's not roll asimov every round on Skyrat

## Changelog

:cl:
balance: overrides asimov with crewsimov
fix: fixed unique AI lawset rolls
config: fixed default configs
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/datum/station_trait/unique_ai
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. To make your life miserable, we have removed your additional lawset boards. Good luck!"
+// Don't roll the default lawsets, please!
+/pick_weighted_lawset()
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: what is this

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	var/datum/ai_laws/lawtype
+	var/list/law_weights = CONFIG_GET(keyed_list/law_weight)
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI))
+		law_weights -= "asimov"
+	while(!lawtype && law_weights.len)
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/roundstart/lawsets.dm]: length better than .len

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 1 above)
Comment (Iamgoofball) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: horrifying abuse of byond

Comment (nevimer) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: override on a root level proc

Comment (nevimer) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: oh you mean the upper part

ok so people keep referring to the build in "Don't change the laws" part is policy when its not, and piss their metaphorical pants about it to the misery of everyone else in the round, as command mains that are also human mains will use it to clutch their pearls at the suggestion it should be changed.

Comment (Iamgoofball) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: There's no point in this trait existing if you're going to remove the blocker on resetting the laws.

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/datum/station_trait/unique_ai
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. To make your life miserable, we have removed your additional lawset boards. Good luck!"
+// Don't roll the default lawsets, please!
+/pick_weighted_lawset()
+	var/datum/ai_laws/lawtype
+	var/list/law_weights = CONFIG_GET(keyed_list/law_weight)
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI))
+		law_weights -= "asimov"
```
Comment (Iamgoofball) [inline modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:8]: take safeguard out of the weights if unique AI is rolled as well

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 1 above)
Comment (nevimer) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: > There's no point in this trait existing if you're going to remove the blocker on resetting the laws.

Not removing the blocker at all, just the suggestion that admins demand the game be played in a certain way

**Comment by YakumoChen** [comment]
Pretty sure speciesism is still codified into the lore... and guiding principles.

Regardless this should be made atomic. Don't mix policy with fixes. That's how we get Goof PRs /s

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 3 above)
Comment (nevimer) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:8]: ok gamer it is done

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 2 above)
Comment (nevimer) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm]: done

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 1 above)
Comment (Iamgoofball) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:4]: i see, carry on

File: modular_skyrat/master_files/code/modules/roundstart/lawsets.dm (hunk 3 above)
Comment (Iamgoofball) [reply modular_skyrat/master_files/code/modules/roundstart/lawsets.dm:8]: god bless

**Pull request closed without merging**